### PR TITLE
Added samples for Protecting Sensitive Data in Event-Sourced Systems with Crypto Shredding

### PR DESCRIPTION
**Pull request by oskardudycz**
# Protecting Sensitive Data in Event-Sourced Systems with Crypto Shredding

This sample is showing an example of using the Crypto Shredding pattern with EventStoreDB. This can be a solution for handling, e.g. European General Data Protection Regulation.

Read more in the Diego Martin article "Protecting Sensitive Data in Event-Sourced Systems with Crypto Shredding".

## Description
- shows how to Protecting Sensitive Data (e.g. for [European General Data Protection Regulation](https://en.wikipedia.org/wiki/General_Data_Protection_Regulation)) in Event-Sourced Systems.
- shows how to use the .NET `System.Security.Cryptography` library with [AES](https://en.wikipedia.org/wiki/Advanced_Encryption_Standard) algorithm to encrypt and decrypt events' data.
- uses EventStoreDB.

## Other changes:
Updated the CI configuring for showing test results to:
- not override each other if there are other changes
- be able to be run by external PR.
Unfortunately, that part is only to be verified once it's merged because of how GH is running the workflow triggers (the workflow file has to be on the main branch already).

### CODE REVIEW

File: CQRS_Flow/.NET/README.md [hunk 1]
```diff
@@ -6,24 +6,22 @@ This sample is showing a  typical flow of the Event Sourcing pattern with [Event
...
+1. Install .NET Core 5.0 - https://dotnet.microsoft.com/download/dotnet/5.0.
+2. Install Visual Studio 2019, Rider or VSCode.
+3. Install docker - https://docs.docker.com/docker-for-windows/install/.
+4. Open `ECommerce.sln` solution.
 
 ## Running
 
 1. Run: `docker-compose up`.
-2. Wait until all dockers got are downloaded and running.
+2. Wait until all dockers are up and running.
```
Comment (alexeyzimarev) [inline CQRS_Flow/.NET/README.md:17]:
```suggestion
2. Wait until all Docker containers are up and running.
```

File: CQRS_Flow/.NET/README.md [hunk 2]
```diff
@@ -6,24 +6,22 @@ This sample is showing a  typical flow of the Event Sourcing pattern with [Event
...
 
-1. Install git - https://git-scm.com/downloads.
-2. Install .NET Core 5.0 - https://dotnet.microsoft.com/download/dotnet/5.0.
-3. Install Visual Studio 2019, Rider or VSCode.
-4. Install docker - https://docs.docker.com/docker-for-windows/install/.
-5. Open `ECommerce.sln` solution.
+1. Install .NET Core 5.0 - https://dotnet.microsoft.com/download/dotnet/5.0.
+2. Install Visual Studio 2019, Rider or VSCode.
+3. Install docker - https://docs.docker.com/docker-for-windows/install/.
+4. Open `ECommerce.sln` solution.
```
Comment (alexeyzimarev) [inline CQRS_Flow/.NET/README.md:12]:
```suggestion
- .NET 5.0 - https://dotnet.microsoft.com/download/dotnet/5.0.
- Visual Studio 2019, Jetbrains Rider or VSCode.
- Docker - https://docs.docker.com/docker-for-windows/install/.
```

File: CQRS_Flow/.NET/README.md [hunk 3]
```diff
@@ -6,24 +6,22 @@ This sample is showing a  typical flow of the Event Sourcing pattern with [Event
...
 
 1. Run: `docker-compose up`.
-2. Wait until all dockers got are downloaded and running.
+2. Wait until all dockers are up and running.
 3. You should automatically get:
     - EventStoreDB UI: http://localhost:2113/
     - ElasticSearch running at http://localhost:9200
     - Kibana - UI for ElasticSearch . Available at: http://localhost:5601
 4. Open, build and run `ECommerce.sln` solution.
 	- Swagger should be available at: http://localhost:5000/index.html
```
Comment (alexeyzimarev) [inline CQRS_Flow/.NET/README.md:23]:
```suggestion
3. Check that you can access each started component the following URLs:
    - EventStoreDB: http://localhost:2113/ (use the default user `admin` and password `changeit`)
    - ElasticSearch: http://localhost:9200
    - Kibana: http://localhost:5601
4. Open, build and run `ECommerce.sln` solution.
    - The Swagger UI should be available at: http://localhost:5000/index.html
```

**Review by alexeyzimarev — CHANGES_REQUESTED** [review]

**Comment by oskardudycz** [comment]
@alexeyzimarev, applied your comments in https://github.com/EventStore/samples/pull/8/commits/8273d1bf9ff808bf23421ea30594e57969a2c2de.

**Review by alexeyzimarev — APPROVED** [review]

**Review by ylorph — APPROVED** [review]

**Pull request merged**